apps/devicepassthrough_apps.py

```diff
@@ -62,8 +74,15 @@ def add_dp_rules(
                'productid': productid, 'vendorid': vendorid}
     configuration.append(newrule)
     dpconfig['devices'] = configuration
+    # If "enable" key not found, set default true
+    if enabled == None:
+        dpconfig['enable'] = True
```

Comment:
No need to implicitly enable it. May be user wants to add only rules, but not enable it

Reply:
I think we can set `enable` key to `False` and print a message as below as instruction for user to enable it.
This is because when generate the json config for ddp (see below `write_vmddp_config`), I check for `enable` key. if it is `false`, then it will not be included in generation. So there should be that key, either `False` or `True`, but it should not be missing.

Reply:
If it should be then it need to be set as False by default, not True